main.cpp

```diff
@@ -63,7 +63,7 @@ GLFWwindow* SetupGLFW()
     glfwMakeContextCurrent(window);
     glfwSetKeyCallback(window, Input::KeyCallback);
 
-    // Uncomment this if you would like to blow up the GPU :D
+    //Uncomment this if you would like to blow up the GPU :D
```

Comment:
🤣

Reply:
Hah yeah. It is cool to uncomment this because it will uncap the fps limit. Normally opengl is tied to whatever the refresh rate of the monitor is. Whenever I uncap it though my GPU usage instantly jumps to and stays at 100% lol